smash/core/model/model.py

```diff
@@ -193,8 +193,8 @@ class Model:
 
         adjust_interception : `bool`, default True
             Whether or not to adjust the maximum capacity of the interception reservoir.
-            This option is only applicable if **hydrological_module** is set to ``'gr4'`` or ``'gr5'`` and
-            for a sub-daily simulation time step **dt**.
+            This option is available for any **hydrological_module** having the :math:`c_i` parameter
+            (i.e. ``'gr4'``, ``'gr5'``, ...) and for a sub-daily simulation time step **dt**.
```

Comment:
and why not enumeratig gr6 too?

Reply:
@pag13, il y a aussi toute les variantes gr4 mlp et ode donc j'en ai listé que 2. Je peux rajouter gr6 tout de meme ;)